app/assets/stylesheets/forever_style_guide/modules/_nav-dropdowns-help.scss

```diff
@@ -0,0 +1,24 @@
+$supporting-text-icon-offset: 33px;
+.nav-dropdown-help {
+  .icon-offset {
+    display: block;
+    padding-left: $supporting-text-icon-offset;
+  }
+}
+.dropdown-menu-help {
+  width: 100%;
```

Comment:
Width: 100% is a code smell. It doesn't look like this or the subsequent right: 0 declaration are necessary?